style.json

```diff
@@ -1241,10 +1256,25 @@
       }
     },
     {
-      "id": "boundary_2",
+      "id": "boundary_2_z0-4",
+      "type": "line",
+      "source": "openmaptiles",
+      "source-layer": "boundary",
+      "maxzoom": 5,
```

Comment:
Is maxzoom: 5 correct here?

Reply:
Yes, there is different source of boundary data in z0-4 (Natural Earth) and then starting z5 (OSM).

Reply:
Ah I see, thanks for the explanation.